### PR TITLE
Load hubspot script and call identity at registration

### PR DESCRIPTION
**Pull request by tejohnso**
## Description
Loads hubspot tracking script from `js.hs-scripts.com/` and then calls the identify action through the script. This is to ensure that users are created in hubspot via the script, and any previous activity from a previous risevision.com session is captured in their hubspot user data.

## Motivation and Context
Improved functionality for marketing.  See [doc](https://drive.google.com/open?id=1xGOVuslmtSXk5VZuPbZDRHPwM6AUZ_JZ4so592cJocM)

## How Has This Been Tested?
E2E + manual testing to confirm user is created in hubspot and updated with company id via backend.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by tejohnso** [comment]
@ezequielc this is the change for the identity call that we discussed on Thursday

File: web/scripts/components/hubspot/svc-hubspot.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        var _hsq = window._hsq = window._hsq || [];
+
+        service.loadAs = function (email) {
+          if (!loaded) {
+             _hsq.push(['identify',{ email: email }]);
+
+            var e = document.createElement('script');
+            e.type = 'text/javascript';
+            e.defer = !0;
+            e.async = !0;
```
Comment (santiagonoguez) [inline web/scripts/components/hubspot/svc-hubspot.js]: curious, why put !0 instead of true ?

File: web/scripts/components/hubspot/svc-hubspot.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  'use strict';
+
+  angular.module('risevision.common.components.hubspot', [])
+
+    .factory('hubspot', ['$rootScope', '$window', 'userState', 'HUBSPOT_ACCOUNT',
+      function ($rootScope, $window, userState, hubspotAccount) {
+        var service = {};
+        var loaded;
+
+        var _hsq = window._hsq = window._hsq || [];
```
Comment (santiagonoguez) [inline web/scripts/components/hubspot/svc-hubspot.js]: what does _hsq stands for ? could we use a clear name here ?

File: web/scripts/components/hubspot/svc-hubspot.js (hunk 1 above)
Comment (tejohnso) [reply web/scripts/components/hubspot/svc-hubspot.js]: Following the style used here `https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/components/segment-analytics/svc-segment-analytics.js#L71`

File: web/scripts/components/hubspot/svc-hubspot.js (hunk 2 above)
Comment (tejohnso) [reply web/scripts/components/hubspot/svc-hubspot.js]: This comes from their example [here](https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/components/segment-analytics/svc-segment-analytics.js#L71). The script file expects that variable name.

File: web/scripts/components/hubspot/svc-hubspot.js (hunk 1 above)
Comment (santiagonoguez) [reply web/scripts/components/hubspot/svc-hubspot.js]: weird, that programmer seems to code in minified mode, hahahahaha, but I'm not against that.

File: web/scripts/components/hubspot/svc-hubspot.js (hunk 2 above)
Comment (santiagonoguez) [reply web/scripts/components/hubspot/svc-hubspot.js]: I see, not nice to mess with window properties, but if that's what the injected script needs, then there's nothing to do about that I guess.

File: web/scripts/components/hubspot/svc-hubspot.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+        service.loadAs = function (email) {
```
Comment (ezequielc) [inline web/scripts/components/hubspot/svc-hubspot.js]: @tejohnso Just nitpicking here, but I would rename this function to `identify()` to match the Hubspot call. Thanks

Comment (tejohnso) [reply web/scripts/components/hubspot/svc-hubspot.js]: It doesn't just make the hubspot call, it pre-configures the hsq variable with the identity call, then it **loads** the hubspot script and the script makes the call.  That's why it's called **load**As

**Comment by tejohnso** [comment]
> @tejohnso Code and functionality look good. I am wondering if you could add unit tests for the `hubspot` service. Thanks!

Unit test added.  Confirms script loads and identity is configured in hsq.  Also added a check to make sure it only loads once, just to get the :100:

**Pull request merged**